be/src/olap/rowset/segment_v2/column_reader.h

```diff
@@ -298,10 +298,10 @@ class FileColumnIterator final : public ColumnIterator {
     ParsedPage _page;
 
     // keep dict page decoder
-    std::unique_ptr<PageDecoder> _dict_decoder;
+    std::shared_ptr<BinaryPlainPageDecoder> _dict_decoder;
 
     // keep dict page handle to avoid released
-    PageHandle _dict_page_handle;
+    std::shared_ptr<PageHandle> _dict_page_handle;
```

Comment:
Could this variable be deleted safely? I notice that this variable will be set to a new value each time so I think it's unnecessary to use as a member variable.